puppeteer/bundleRequests.test.js

```diff
@@ -81,13 +81,19 @@ describe('Js bundle requests', () => {
               const serviceMatches = requests.filter(url =>
                 url.match(
                   new RegExp(
-                    `(\\/static\\/js\\/${config[service].name}-.+?.js)`,
+                    `(\\/static\\/js\\/${config[service].name}.+?.js)`,
                     'g',
                   ),
                 ),
               );
 
-              expect(serviceMatches.length).toBeGreaterThanOrEqual(1);
+              if (config[service].name === 'russian') {
```

Comment:
if we want the russian service config to be separated out of shared then I can uncomment these lines https://github.com/bbc/simorgh/pull/9255/files#diff-237bfab1f7bb16a90027e7bb26a4bf029a246977dafe45aba360292963261fa1R149-R156 however the total bundle size jumps by around 20-25k as does the ukrainian service bundle

Reply:
Any idea why the bundlesize grows? I'd expect it decrease across the board if russian was extracted out?

Reply:
I think because the ukrainian service bundle needs the russian service config inside of it which means the ukrainian service bundle is double the size of other service bundles. webpack realises russian is better off in a shared chunk. let me know if you want a call to go through that and show you the bundles.